package.json

```diff
@@ -29,11 +29,15 @@
     "docs:build": "npx typedoc",
     "docs:build:watch": "npm run docs:build -- --watch",
     "docs:serve": "PORT=4000 npx serve --no-clipboard docs",
-    "test": "npx jest",
-    "test:ci": "npm run test -- --ci --verbose false",
-    "test:watch": "npm run test -- --watchAll",
-    "test:e2e": "npm run test -- --config __tests__/e2e/jest.e2e-config.js",
-    "test:e2e:watch": "npm run test -- --config __tests__/e2e/jest.e2e-config.js --watchAll",
+    "test:unit": "npx jest",
+    "test:ci": "npm run test:unit -- --ci --verbose false && npm run test:e2e -- --ci --verbose false",
+    "test:watch": "npx jest -- --watchAll",
+    "test:pack-library": "npm pack --json --pack-destination ./__tests__/e2e/import-library/export && mv __tests__/e2e/import-library/export/onebeyond-agnostic-cms-harmonizer-$npm_package_version.tgz __tests__/e2e/import-library/export/package.tgz",
```

Comment:
Suspicious `$` in filename. Please confirm!
<code>mv __tests__/e2e/import-library/export/onebeyond-agnostic-cms-harmonizer-<em><strong>$</strong></em>npm_package_version.tgz
</code>

Reply:
Hi @matyasjay , the $ is needed because the `$npm_package_version` is a environment variable that npm provide you so this way we make sure that we are using the right file

Reply:
Cool, I thought something like that. Thanks for confirming!